model-optimizer/mo/utils/cli_parser.py

```diff
@@ -1127,6 +1130,85 @@ def get_absolute_path(path_to_file: str) -> str:
     return file_path
 
 
+def isfloat(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def detect_value_type(value: str):
+    values = value.split(',')
+    for i in range(len(values)):
+        value = values[i]
+        if value.isdigit():
+            values[i] = int(value)
+        elif isfloat(value):
+            values[i] = float(value)
+
+    return values[0] if len(values) == 1 else values
+
+
+def parse_transform(transform: str, ie_is_available=True) -> list:
```

Comment:
`ie_is_available` should not be passed to this function. The main purpose of this function should be just to parse the string with the transformations information.

Reply:
agree. changed.